File: DungeonDebt/Assets/Scripts/Data/RelicDefinition.cs
public class RelicDefinition
{
    public RelicDefinition(RelicId id, string displayName, string effectDescription)
    {
        Id = id;
        DisplayName = displayName;
        EffectDescription = effectDescription;
    }

    public RelicId Id { get; }
    public string DisplayName { get; }
    public string EffectDescription { get; }
}

File: web/src/data/RelicDefinition.js
// Ported from DungeonDebt/Assets/Scripts/Data/RelicDefinition.cs
export class RelicDefinition {
  constructor(id, displayName, effectDescription) {
    this.id = id;
    this.displayName = displayName;
    this.effectDescription = effectDescription;
    Object.freeze(this);
  }
}
